package.json

```diff
@@ -5,6 +5,7 @@
   "devDependencies": {
     "autoprefixer": "^9.7.6",
     "docsy": "github:google/docsy#semver:0.6.0",
+    "@fortawesome/fontawesome-free": "^6.7.2",
```

Comment:
Do you think that this can be removed as updating fontawesome doesn't help?

Reply:
Might as well, removed

Reply:
Apologies for the nitpick, but the lock file change should be reverted too.